container.go

```diff
@@ -120,6 +120,43 @@ func (c *Container) URL() string {
 	return c.pod.URL()
 }
 
+func (c *Container) startShim() error {
+	proxyInfo, url, err := c.pod.proxy.connect(*(c.pod), true)
+	if err != nil {
+		return err
+	}
+
+	if err := c.pod.proxy.disconnect(); err != nil {
+		return err
+	}
+
+	if c.pod.state.URL != url {
+		return fmt.Errorf("Pod URL %s and URL from proxy %s MUST be similar", c.pod.state.URL, url)
```

Comment:
Shouldn't this be "identical" rather than "similar" (same comment for identical error below)?